test/common/test/api_utils_test.py

```diff
@@ -0,0 +1,249 @@
+"""
+Copyright 2017, EMC, Inc.
+"""
+import sys
+import subprocess
+
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test")
+
+import unittest
+
+#CIT
+from on_http_api2_0 import ApiApi as Api
+from config.api1_1_config import config as config_old   #TODO remove when 2.0 worklfow API is implemented
+from config.api2_0_config import config
+from json import loads
+
+from common.api_utils import api_node_select_from_config, api_node_select, api_validate_node_pollers
+
+from nose.plugins.attrib import attr
+@attr(all=True, regression=True, smoke=True)
+
+class NodesTests(unittest.TestCase):
+
+    def setUp(self):
+        self.__client_old = config_old.api_client
+        self.__client = config.api_client
+    def __get_data(self):
+        return loads(self.__client.last_response.data)
+
+    def test_type_list(self):
+        """
+        Test of the api_node_select() function
```

Comment:
How does this show up when you run this? I know a one-line doc-string will augment what gets shown in the 'nosetests -v' output. I'm not sure what a multi-line one does?

"validate test utility method api_node_select() output per node_type matches raw nodes_get_all() data"

is what I was aiming for. Sorry I wasn't more clear...